source/WindowScroller/utils/onScroll.js

```diff
@@ -1,3 +1,9 @@
+import requestAnimationTimeout, {
+  cancelAnimationTimeout
+} from '../../utils/requestAnimationTimeout'
```

Comment:
Looks a bit weird. Maybe better to use only named exports? And then filename can be changed to `animationTimeout.js`